packages/headless/src/features/generated-answer/generated-answer-request.ts

```diff
@@ -132,7 +145,7 @@ export const constructAnswerAPIQueryParams = (
     ...(state.debug !== undefined && {debug: state.debug}),
     referrer,
     ...actionsHistory,
-    ...(foldingParams && foldingParams),
+    ...foldingParams,
```

Comment:
Why this change? this could potentially be breaking in the case of `foldingParams` is undefined.